polkadot-parachains/westmint/src/lib.rs

```diff
@@ -702,7 +700,6 @@ construct_runtime!(
 		ParachainSystem: cumulus_pallet_parachain_system::{
 			Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned,
 		},
-		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage},
```

Comment:
we cannot just remove this, it would change the indices. You'll need to add an explicit index at least for the timestamp pallet directly after.

Reply:
Talked with Joe about it and we want to align the indices between Westmint and Statemint (https://github.com/paritytech/cumulus/issues/729) so this is less relevant I guess. We'll just need to bump the tx_version at some point

Reply:
Can I align them in this PR?

Reply:
I'm opening a separate PR

Reply:
If you bump the tx_version in #731 I think this one can be merged

Reply:
Or I bump it here?